lib/csv_row_model/model/columns.rb

```diff
@@ -49,6 +61,7 @@ def _columns
         # @option options [Hash] :type class you want to automatically parse to (by default does nothing, equivalent to String)
         # @option options [Hash] :parse a Proc for parsing the cell
         # @option options [Hash] :default default value of the column if it is blank?, can pass Proc
+        # @option options [Hash] :friendly_name human friendly string of the column name, by default column_name
```

Comment:
I added this so we are able to have a frindely string to sport that's different than the name. According to the following link the archives with export can have a very different name to the column _name https://docs.google.com/spreadsheets/d/1atannMKoUmMeWfIiqi195Z4ZHtVS1TuRxY43ZrTKlsg/edit?usp=sharing